ui/nuxeo-path-suggestion/nuxeo-path-suggestion.js

```diff
@@ -108,7 +107,7 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
           value="{{value}}"
           data="[[data]]"
           allowed-pattern="[[allowedPattern]]"
-          auto-validate="[[autoValidate]]"
```

Comment:
@andreacornaglia we were directly passing the `autoValidate` here, which allowed the `paper-typeahead` to trigger the validation automatically. But based on the behavior I described, it makes much more sense to do the validation here (and trigger it, if `autoValidate` is set to true).